### PR TITLE
how-to/deploy: add code snippets and improve description

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? 

- Add code snippets for `ansible.md` to make it more friendly and easier to use
- Improve description, organization (using lists)
- Change `python-pip` to `python2-pip`

### What is the related PR or file link(s)? 

N/A

### Which version does your change affect? 

dev, v3.1, v3.0, v2.1

### CODE REVIEW

File: dev/how-to/deploy/orchestrated/ansible.md [hunk 1]
```diff
@@ -59,15 +59,19 @@ Log in to the Control Machine using the `root` user account, and run the corresp
...
 - If you use a Control Machine installed with CentOS 7, run the following command:
 
-    ```
-    # yum -y install epel-release git curl sshpass
-    # yum -y install python-pip
+    {{< copyable "shell-root" >}}
+
+    ```shell
+    yum -y install epel-release git curl sshpass && \
+    yum -y install python2-pip
```
Comment (lilin90) [inline dev/how-to/deploy/orchestrated/ansible.md]: @liubo0127 I modified `python-pip` to `python2-pip` for all docs versions according to [that in the Chinese version](https://pingcap.com/docs-cn/stable/how-to/deploy/orchestrated/ansible/#%E5%9C%A8%E4%B8%AD%E6%8E%A7%E6%9C%BA%E4%B8%8A%E5%AE%89%E8%A3%85%E7%B3%BB%E7%BB%9F%E4%BE%9D%E8%B5%96%E5%8C%85).

File: dev/how-to/deploy/orchestrated/ansible.md [hunk 2]
```diff
@@ -59,15 +59,19 @@ Log in to the Control Machine using the `root` user account, and run the corresp
...
     ```
 
 - If you use a Control Machine installed with Ubuntu, run the following command:
 
-    ```
-    # apt-get -y install git curl sshpass python-pip
+    {{< copyable "shell-root" >}}
+
+    ```shell
+    apt-get -y install git curl sshpass python-pip
```
Comment (lilin90) [inline dev/how-to/deploy/orchestrated/ansible.md]: @liubo0127 Do we need to change `python-pip` to `python2-pip` for this line, too?

File: dev/how-to/deploy/orchestrated/ansible.md [hunk 3]
```diff
@@ -765,109 +859,125 @@ Edit the `inventory.ini` file and add the following host variable after the IP o
...
+    ```shell
+    python
+    ```
+
+    ```
+    Python 2.7.5 (default, Nov  6 2016, 00:28:07)
+    [GCC 4.8.5 20150623 (Red Hat 4.8.5-11)] on linux2
+    Type "help", "copyright", "credits" or "license" for more information.
+    >>> import jmespath
+    ```
```
Comment (lilin90) [inline dev/how-to/deploy/orchestrated/ansible.md]: @liubo0127 Is the above code block correct in the way we put it? Could it be the following version?

{{< copyable "shell-regular" >}}

```shell
python
```

```
Python 2.7.5 (default, Nov  6 2016, 00:28:07)
[GCC 4.8.5 20150623 (Red Hat 4.8.5-11)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
```

{{< copyable "shell-regular" >}}

```shell
import jmespath
```

File: dev/how-to/deploy/orchestrated/ansible.md [hunk 4]
```diff
@@ -765,109 +859,125 @@ Edit the `inventory.ini` file and add the following host variable after the IP o
...
-```
+3. Enter `import jmespath` in the Python interactive window of the Control Machine.
+
+    - If no error displays, the dependency is successfully installed.
+    - If the `ImportError: No module named jmespath` error displays, the Python `jmespath` module is not successfully installed.
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    python
```
Comment (lilin90) [inline dev/how-to/deploy/orchestrated/ansible.md]: @liubo0127 Is running the `python` command here a necessary step? Why?

File: dev/how-to/deploy/orchestrated/ansible.md (hunk 1 above)
Comment (liubo0127) [reply dev/how-to/deploy/orchestrated/ansible.md]: LGTM

File: dev/how-to/deploy/orchestrated/ansible.md (hunk 2 above)
Comment (liubo0127) [reply dev/how-to/deploy/orchestrated/ansible.md]: It's `python-pip` on Ubuntu operation system.

File: dev/how-to/deploy/orchestrated/ansible.md [hunk 5]
```diff
@@ -765,109 +859,125 @@ Edit the `inventory.ini` file and add the following host variable after the IP o
...
 ```shell
-# process supervision, [systemd, supervise]
+process supervision, [systemd, supervise]
+```
+
+```
 process_supervision = systemd
 ```
 
 For versions earlier than TiDB 1.0.4, the TiDB Ansible supervision method of a process is `supervise` by default. The previously installed cluster can remain the same. If you need to change the supervision method to `systemd`, stop the cluster and run the following command:
```
[the remark this answers is not in the document]
Comment (liubo0127) [reply dev/how-to/deploy/orchestrated/ansible.md]: I think It does not need to changed.

File: dev/how-to/deploy/orchestrated/ansible.md (hunk 3 above)
Comment (liubo0127) [reply dev/how-to/deploy/orchestrated/ansible.md]: It is better to write separately according to comment.

File: dev/how-to/deploy/orchestrated/ansible.md (hunk 4 above)
Comment (liubo0127) [reply dev/how-to/deploy/orchestrated/ansible.md]: Yes, this step confirms whether the dependent package can be used normally.

File: dev/how-to/deploy/orchestrated/ansible.md (hunk 2 above)
Comment (lilin90) [reply dev/how-to/deploy/orchestrated/ansible.md]: Got it. Then let's keep the existing version.

File: dev/how-to/deploy/orchestrated/ansible.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (lilin90) [reply dev/how-to/deploy/orchestrated/ansible.md]: OK. Then let's keep it for now.

File: dev/how-to/deploy/orchestrated/ansible.md (hunk 4 above)
Comment (lilin90) [reply dev/how-to/deploy/orchestrated/ansible.md]: Got it. Thanks!

File: dev/how-to/deploy/orchestrated/ansible.md (hunk 3 above)
Comment (lilin90) [reply dev/how-to/deploy/orchestrated/ansible.md]: Then I'll use the separated version.

**Comment by lilin90** [comment]
@liubo0127 Comments addressed. PTAL again~

**Pull request merged**